blips/blip-0001.md

```diff
@@ -0,0 +1,289 @@
+are publicly broadcasted on the Lightning Network. Feature Bits are first introduced as
+optional with *odd* bits, and when upgraded to compulsory are switched to *even* bits, 
+so a new feature proposed in a bLIP should start with an *odd* bit. Feature Bits are 
+assigned and specified in [Bolt
+ #9](https://github.com/lightningnetwork/lightning-rfc/blob/master/09-features.md).
+* **Message Types** are used to indicate how to interpret the `payload` feature of a 
+Lightning message. Types `32768`-`65535` are set aside for experimental and 
+application-specific messages, which are best suited to be documented in a bLIP. 
+Message Types are assigned and specified in [Bolt
+ #1](https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md).
+* **Type-Length-Values (TLVs)** are used to allow for the backwards-compatible addition 
+of new fields to existing message types. `type` determines how the contents of `value`
+should be decoded, so specifying how to actually perform that decoding is a
+great use case for a bLIP. TLVs are not assigned anywhere, but specified in [Bolt
+ #1](https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md).
+
+bLIPs will serve as the primary mechanism for proposing new features for the Lightning 
+Network protocol, documenting their design, and avoiding collisions of these scarce 
+identifiers (as some proposals may request one or more). Hopefully, they will provide 
+an avenue for developers to quickly get feedback on their ideas outside of the main BOLT 
+process. Because the bLIPs are maintained as text files in a versioned repository, 
+their revision history is the historical record of the feature proposal.
+
+It is highly recommended that a single bLIP contain a single key proposal or new idea. 
+More focused bLIPs will tend to be more successful. If in doubt, a bLIP should be 
+split into several well-focused ones.
+
+For Lightning developers, bLIPs are a convenient way to track the progress of their 
+implementation. Ideally, each implementation editor would list the bLIPs they have 
+implemented. This will give end users a convenient way to know the current status of 
+a given implementation or library.
+
+# bLIP Workflow
+
+The bLIP process begins with a new idea for Lightning. Each potential bLIP must have 
+a champion -- someone who writes the bLIP using the style and format described below, 
+shepherds the discussions in the appropriate forums, and attempts to build community 
+consensus around the idea. The bLIP champion (a.k.a. Author) should first attempt to 
+ascertain whether the idea is bLIP-able. The first step should be to search past 
+discussions to see if an idea has been considered before, and if so, what issues arose 
+in its progression. Such discussion generally happens on the [Lightning development 
+mailing list](https://lists.linuxfoundation.org/mailman/listinfo/lightning-dev), or 
+in the #lightning-dev IRC channel.
```

Comment:
Maybe also recommend to check if feature X has been not already proposed in the BIP namespace or dlcspec namespace?

Reply:
good idea, added!